.python-version

```diff
@@ -0,0 +1,4 @@
+2.6.9
```

Comment:
We don't need this, I think, but an instruction how we use `pyenv` and `pyenv local` instead.